.travis.yml

```diff
@@ -41,3 +41,14 @@ env:
 
 script:
   - ./verbose-test
+  - cd docs && make html
+
+deploy:
+  provider: pages
+  skip_cleanup: true
+  keep-history: true
+  github_token: $GITHUB_TOKEN
+  on:
+    branch: travis-sphinx-doc
```

Comment:
This needs to be updated to either `3.0beta` or `master`. `master` should be the correct branch since we only want to generate new documentation whenever an update goes into master, but if that's the case then we may not see any new updates until `3.0beta` is merged into `master`.